packages/editor/__tests__/operation/__snapshots__/adjustComponentOrder.spec.ts.snap

```diff
@@ -0,0 +1,2725 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@Sczlog  @tanbowensg 

I don't think the current size of snapshots is reviewable. Could we use a tiny but comprehensive fixture and only use small inline snapshots when needed. Otherwise, we should write other assertions to test the expected behavior.